txpool/tx_object_test.go

```diff
@@ -61,21 +59,21 @@ func txBuilder(txType tx.Type, chainTag byte, clauses []*tx.Clause, gas uint64,
 		Gas(gas)
 }
 
-func SetupTest() (genesis.DevAccount, *chain.Repository, *block.Block, *state.State) {
-	acc := genesis.DevAccounts()[0]
+func SetupTest() (genesis.DevAccount, *chain.Repository, *block.Block, *state.State, thor.ForkConfig) {
+	tchain, _ := testchain.NewWithFork(thor.SoloFork)
+	repo := tchain.Repo()
+	db := tchain.Database()
 
-	db := muxdb.NewMem()
-	repo := newChainRepo(db)
 	b0 := repo.GenesisBlock()
 	b1 := new(block.Builder).ParentID(b0.Header().ID()).GasLimit(10000000).TotalScore(100).Build()
 	repo.AddBlock(b1, nil, 0, false)
 	st := state.New(db, trie.Root{Hash: repo.GenesisBlock().Header().StateRoot()})
 
-	return acc, repo, b1, st
+	return genesis.DevAccounts()[0], tchain.Repo(), b1, st, tchain.GetForkConfig()
 }
 
 func TestExecutableWithError(t *testing.T) {
-	acc, repo, b1, st := SetupTest()
+	acc, repo, b1, st, fc := SetupTest()
```

Comment:
This test is unexpected it's called `TestExecutableWithError` but the test cases don't have errors

Reply:
https://github.com/vechain/thor/pull/1070/files/db28a767bcc5b851a10c3cefc258cc16cd9d5173#diff-fd5ec3edcfcaf37883d0def12c40307c2ec636ea22335c855991254de9ad2872R94 is asserting errors